Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/GroupListPickerViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  GroupListPickerViewController.swift
+//  StreamChatUI
+//
+//  Created by Parth Kshatriya on 08/04/22.
+//
+
+import UIKit
+import StreamChat
+import Nuke
+
+class GroupListPickerViewController: UIViewController {
```

Comment:
@mdhanif-simformsolutions  Pls remove these controller if not required.

Reply:
Removed this controller as it is not required for now.